tests/formats_test/misc_test.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+
+from blivet.formats.fs import *
+from blivet.size import Size
+
+class FSOverheadTestCase(unittest.TestCase):
+
+    def testRequiredSizeFS(self):
+        # FS is abstract parent which doesn't have metadata
+        self.assertEqual(FS.getRequiredSize(Size("100 MiB")), Size("100 MiB"))
+        self.assertEqual(Ext2FS.getRequiredSize(Size("100 B")), Size("107 B"))
```

Comment:
I doubt an ext2 file system this big could ever be created. ;) Maybe change the unit to ``MiB``?